src/Raven.Server/Documents/Subscriptions/SubscriptionFetcher.cs

```diff
@@ -96,26 +98,26 @@ public RevisionSubscriptionFetcher(DocumentDatabase database, SubscriptionConnec
 
         }
 
-        protected override IEnumerable<(Document Previous, Document Current)> FetchByEtag()
+        protected override IEnumerator<(Document Previous, Document Current)> FetchByEtag()
```

Comment:
Enumerator has Dispose method, are we calling that somewhere?

Reply:
done